index.js

```diff
@@ -63,11 +64,21 @@ function pixelmatch(img1, img2, output, width, height, options) {
                                            antialiased(img2, x, y, width, height, img1))) {
                     // one of the pixels is anti-aliasing; draw as yellow and do not count as difference
                     // note that we do not include such pixels in a mask
-                    if (output && !options.diffMask) drawPixel(output, pos, aaR, aaG, aaB);
+                    if (output && !options.diffMask) drawPixel(output, pos, ...options.aaColor);
```

Comment:
This is a hot path (done for every pixel) so let's be sure to measure if the spread has a performance overhead.

Reply:
Not sure if this is enough but I don't believe I am seeing any new overhead. Running pixelmatch [against these images](https://github.com/mapbox/pixelmatch/files/4488844/Archive.zip) a few times I get close to these milliseconds:


- Branch `issue-86`

<img width="167" alt="issue-86" src="https://user-images.githubusercontent.com/61150/79487887-ed613d80-7fe6-11ea-9bc9-125c92ad4187.png">

- Branch `master`

<img width="167" alt="master" src="https://user-images.githubusercontent.com/61150/79488189-59dc3c80-7fe7-11ea-8df7-a9e57fa76e94.png">